### DISCUSSION TITLE
Add support for updating Maven extensions

### DISCUSSION THREAD

**Discussion by breun** [renovatebot/renovate#29279] [Suggest an Idea]
### Tell us more.

Project-specific [Maven extensions](https://maven.apache.org/guides/mini/guide-using-extensions.html) can be defined in `.mvn/extensions.xml`. Currently Renovate doesn't create pull requests for these extensions.

For example, consider a project with this `.mvn/extensions.xml` file:

```
<?xml version="1.0" encoding="UTF-8"?>
<extensions>
  <extension>
    <groupId>io.takari.maven</groupId>
    <artifactId>takari-smart-builder</artifactId>
    <version>0.6.3</version>
  </extension>
</extensions>
```

Ideally Renovate would create a pull request to update this extension to the latest version ([currently 0.6.6 is on Maven Central](https://central.sonatype.com/artifact/io.takari.maven/takari-smart-builder)).

**Comment by rarkins** [comment]
Should already be supported since https://github.com/renovatebot/renovate/pull/28893

**Comment by breun** [comment]
Due to an issue on my side I was using a version of Renovate that was approximately one month old. But great to hear it's already been implemented in the meantime!
